### PR TITLE
clarify aspect ratio processing

### LINKED ISSUE #40: Track width, height and aspect ratio
The current AV1-ISOBMFF spec says:
> The width and height fields of the VisualSampleEntry SHALL equal the values of max_frame_width_minus_1 + 1 and max_frame_height_minus_1 + 1 of the Sequence Header OBU applying to the samples associated with this sample entry.

> As specified in [ISOBMFF], the width and height in the VisualSampleEntry are specified in square pixels. If the video pixels are not square, then a pasp box SHALL be included and the track header width and height SHOULD match the values of max_frame_width_minus_1 + 1 and max_frame_height_minus_1 + 1 after the pasp ratio has been applied.

However the AV1 spec has an additional notion of render_width:
>The render size is provided as a hint to the application about the desired display size. It has no effect on the decoding process.

Note that like frame width and height, this can change from frame to frame, and there is not max_render_width/height.

Should we keep the current text or change it to say: 
- track width and height should (or maybe shall) be set to the max render width and height (which defaults to max_frame_width/height if render width/height is not used);
- pasp shall be included if render aspect ratio differs from frame aspect ratio and shall match the difference?

### PR DESCRIPTION
**Pull request by cconcolato**
Closes #40

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -172,7 +172,12 @@ class AV1SampleEntry extends VisualSampleEntry('av01') {
 
 The <dfn noexport>width</dfn> and <dfn noexport>height</dfn> fields of the [=VisualSampleEntry=] SHALL equal the values of [=max_frame_width_minus_1=] + 1 and [=max_frame_height_minus_1=] + 1 of the [=Sequence Header OBU=] applying to the samples associated with this sample entry.
 
-As specified in [[!ISOBMFF]], the width and height in the [=VisualSampleEntry=] are specified in square pixels. If the video pixels are not square, then a 'pasp' box SHALL be included and the track header width and height SHOULD match the values of [=max_frame_width_minus_1=] + 1 and [=max_frame_height_minus_1=] + 1 after the 'pasp' ratio has been applied.
+
```
Comment (tomfinegan) [inline index.bs]: nit: remove the extra blank line

File: index.bs [hunk 2]
```diff
@@ -172,7 +172,12 @@ class AV1SampleEntry extends VisualSampleEntry('av01') {
...
+The width and height in the TrackHeaderBox SHOULD equal the maximum RenderWidth and RenderHeight values of all the frames associated with this sample entry, respectively called MaxRenderWidth and MaxRenderHeight. Additionally, if MaxRenderWidth and MaxRenderHeight values do not equal respectively the [=max_frame_width_minus_1=] + 1 and [=max_frame_height_minus_1=] + 1 values of the [=Sequence Header OBU=], a PixelAspectRatioBox box SHALL be present in the sample entry and set such that 
+```
+hSpacing/vSpacing = MaxRenderWidth*(max_frame_height_minus_1+1)/
+                        ((max_frame_width_minus_1+1)*MaxRenderHeight).
```
Comment (tomfinegan) [inline index.bs]: How about:

    hSpacing / vSpacing = MaxRenderWidth * (max_frame_height_minus_1 + 1) /
                            ((max_frame_width_minus_1 + 1) * MaxRenderHeight).

I think it's easier to read with the spacing around the operators.

**Comment by cconcolato** [comment]
@tomfinegan how does it look now?

@dwsinger Can you have a look?

**Comment by tomfinegan** [comment]
still LGTM; thanks!

**Comment by dwsinger** [comment]
afaict ok!

**Pull request merged**